docs/scos/dev/front-end-development/202307.0/oryx/building-applications/styling/oryx-icon-system.md

```diff
@@ -146,6 +147,24 @@ export const app = appBuilder().withTheme({
 });
 ```
 
+### Configure icons for right-to-left
+
+Icon libraries are typically created for left-to-right (LTR), but when they're used in right-to-left (RTL) some icons much flip to reflect the layout. This is affecting only icons that express a direction, such as arrow icons.
+
+Oryx supports [directionality](/docs/scos/dev/front-end-development/{{page.version}}/oryx/internationalisation/directionality.html) and to ensure that some icons will flip accordingly, the icon style must be set with the `direction: true` config. The following example shows how the forward icon is configured to flip in RTL mode.
+
+```ts
+export const materialDesignIcons: IconMapper = {
+  id: "material-icons",
+  mapping: {
+    [IconTypes.Forward]: {
+      text: "chevron_right",
+      styles: { direction: true },
```

Comment:
Shouldn't it be `direction: rtl`?

Reply:
Hm, checking an implementation, but it looks a bit weird, we are using `styles`, property which should be used for css only, but passing some configuration.

Reply:
These are actually style configurations, such as font, fill, weight

```ts

export interface IconStyles {
  font?: string;
  fill?: number;
  weight?: number;
  grad?: number;
  optical?: number;
  size?: 'string';
  direction?: boolean;
}
```

Not sure why you think its confusing.

Reply:
For me it was a bit confusing we are using other properties in styles object as css properties, but this one - in a different way. And I was not sure just from configuration what does it mean `direction: true` or `direction: false`. And there is no explanation in interface.

Reply:
understood. I've added an additional explanation to the parent section where these style configurations are introduced.